models/image_classification/squeezenet/README.md

```diff
@@ -0,0 +1,59 @@
+# SqueezeNet
+
+## Use cases
+SqueezeNet models perform image classification - they take images as input and classify the major object in the image into a set of pre-defined classes. They are trained on ImageNet dataset which contains images from 1000 classes. SqueezeNet models are highly efficient in terms of size and speed while providing good accuracies. This makes them ideal for platforms with strict constraints on size.
+
+## Description
+SqueezeNet is a small CNN which achieves AlexNet level accuracy on ImageNet with 50x fewer parameters. SqueezeNet requires less communication across servers during distributed training, less bandwidth to export a new model from the cloud to an autonomous car and more feasible to deploy on FPGAs and other hardware with limited memory.
+
+## Model
+Squeezenet 1.1 gives AlexNet level of accuracy with 50X fewer parameters.
+<!-- SqueezeNet 1.1 has 2.4x less computation and slightly fewer parameters than SqueezeNet 1.0, without sacrificing accuracy.-->
+
+|Model        |Download  | Download (with sample test data)|ONNX Version|Top-1 accuracy (%)|Top-5 accuracy (%)|
```

Comment:
Could you also specify the opset version here? working with which version of onnx?

Reply:
done!